activesupport/test/number_helper_test.rb

```diff
@@ -293,6 +293,8 @@ def test_number_to_human
           assert_equal '1.2346 Million', number_helper.number_to_human(1234567, :precision => 4, :significant => false)
           assert_equal '1,2 Million', number_helper.number_to_human(1234567, :precision => 1, :significant => false, :separator => ',')
           assert_equal '1 Million', number_helper.number_to_human(1234567, :precision => 0, :significant => true, :separator => ',') #significant forced to false
+          assert_equal '1 Million', number_helper.number_to_human(999999)
+          assert_equal '1 Billion', number_helper.number_to_human(999999999)
```

Comment:
I think the assertion is correct and the changelog entry could have been written better